src/test/java/stepDefinition/LoginStepDefinition.java

```diff
@@ -0,0 +1,59 @@
+package stepDefinition;
+
+import io.cucumber.java.en.And;
+import io.cucumber.java.en.Given;
+import io.cucumber.java.en.Then;
+import io.cucumber.java.en.When;
+import io.github.bonigarcia.wdm.WebDriverManager;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.chrome.ChromeOptions;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import org.testng.Assert;
+import pages.LoginPage;
+
+import java.time.Duration;
+
+public class LoginStepDefinition {
+    WebDriver driver;
+    WebDriverWait wait;
+
+    @Given("I open browser")
+    public void openBrowser(){
+        WebDriverManager.chromedriver().setup();
+        ChromeOptions optionsChrome = new ChromeOptions();
+        optionsChrome.addArguments("--remote-allow-origins=*","--disable-notifications","--incognito", "--start-maximized");
+        driver = new ChromeDriver();
```

Comment:
You got this problem, please correct it. 

![image](https://user-images.githubusercontent.com/104761482/226684962-6be2a41c-8591-400e-8b82-b19cae854abd.png)


HINT

`driver = new ChromeDriver(optionsChrome);`